making a new top level section to work with DataFrames (#2717)
JuliaData/DataFrames.jl: docs/src/assets/README.md

# Introduction

In this folder we store the following data sets:

- german_credit.csv
- iris.csv

# German Credit data set

## License:
https://opendatacommons.org/licenses/dbcl/1-0/

## Source:
https://archive.ics.uci.edu/ml/datasets/statlog+(german+credit+data)
Professor Dr. Hans Hofmann
Institut für Statistik und Ökonometrie
Universität Hamburg
FB Wirtschaftswissenschaften
Von-Melle-Park 5
2000 Hamburg 13

> The original data is from [UCI](https://archive.ics.uci.edu/ml/datasets/statlog+(german+credit+data)), and that this data is from [Kaggle](https://www.kaggle.com/uciml/german-credit)

Reviewer: ```suggestion
The original data is from [UCI](https://archive.ics.uci.edu/ml/datasets/statlog+(german+credit+data)),
and the file stored here is from [Kaggle](https://www.kaggle.com/uciml/german-credit)
```
